scss/components/_icon.scss

```diff
@@ -0,0 +1,13 @@
+.icon {
+  &_sm {
```

Comment:
why do you need this modifier if it is only one? you could apply this styles directly to the `icon` component